src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -757,7 +757,9 @@ static unique_ptr<Scene> _sceneBuildHelper(const SvgNode* node, const Box& vBox,
 static void _applySvgViewFlag(const Scene* scene, float& vx, float& vy, float& vw, float& vh, float& w, float& h, SvgViewFlag viewFlag)
 {
     if (!((uint32_t)viewFlag & (uint32_t)SvgViewFlag::Viewbox)) {
-        scene->bounds(nullptr, nullptr, &vw, &vh, false);
+        scene->bounds(&vx, &vy, &vw, &vh, false);
```

Comment:
stroke and viewbox looks having no relavent..
Curious two fixes are in one.

Reply:
right. this change moved to #1334 (with a change)